### PR TITLE
10 docs cronograma milestones

### PR DESCRIPTION
**Pull request by samuel-sato**
Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to <repo_name>?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [X] I have read the CONTRIBUTING.md doc.
- [X] I have added necessary documentation (if appropriate).
- [X] Any dependent changes have been merged and published in downstream modules.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by ananorberto** [comment]
Documento pronto para revisão @Mylena-angelica.

File: docs/Base/cronograma.md [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+    </tr>
+    <tr>
+      <td>BPMN</td>
+      <td>13/09/2023</td>
+      <td>14/09/2023</td>
+      <td>Bruno, Gabriel e Samuel</td>
+      <td>-</td>
+    </tr>
+      <td style="vertical-align: middle" rowspan="1"><h3><b>Decidir</b></h3>(1 dias)</td>
+      <td>Frist things Frist</td>
```
Comment (guilherme-puida) [inline docs/Base/cronograma.md:116]:
```suggestion
      <td>First things First</td>
```

**Review by guilherme-puida — CHANGES_REQUESTED** [review]

File: docs/Base/cronograma.md [hunk 2]
```diff
@@ -0,0 +1,209 @@
+# Cronograma das Entregas
+
+## Introdução
+
+Este documento é referente ao cronograma que será seguido pela equipe durante o semestre de 2023.2 da disciplina de Arquitetura e desenho de Software.
+O mesmo apresenta as datas limites de entrega e revisão, juntamente com os autores e revisores, para cada etapa do desenvolvimento do projeto.
```
Comment (Mylena-angelica) [inline docs/Base/cronograma.md:6]: O mesmo apresenta as datas limites de entrega e revisão, juntamente com os autores para cada etapa do desenvolvimento do projeto. Devido ao fato de ser um projeto em andamento, os autores e revisores estão sujeitos á mudanças.

**Review by Mylena-angelica — CHANGES_REQUESTED** [review]

**Comment by ananorberto** [comment]
Ótimas observações @guilherme-puida e @Mylena-angelica, já faço as alterações.

**Review by Mylena-angelica — APPROVED** [review]
Alterações das reviews feitas. Pronto para merge

**Review by guilherme-puida — APPROVED** [review]
:+1:

**Pull request merged**